sanic/app.py

```diff
@@ -1369,6 +1369,12 @@ async def _websocket_handler(
             protocol = request.transport.get_protocol()
             ws = await protocol.websocket_handshake(request, subprotocols)
 
+        await self.dispatch(
+            "websocket.handler.before",
+            inline=True,
+            context={"request": request, "websocket": ws},
+            fail_not_found=False,
```

Comment:
I don't think we need this.

Reply:
This is the only place to access the WS prior to the handler, and as such is very much needed.